development/file_formats/tscn.rst

```diff
@@ -60,6 +60,190 @@ so on. For example, a spatial node looks like:
     [node name="Cube" type="Spatial" parent="."]
     transform=Transform( 1.0, 0.0, 0.0 ,0.0, 1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0 )
 
+The Scene Tree
+--------------
+
+The scene tree is made up of ... nodes! The heading of each node consists of
+it's name, parent and (most of the time) a type. For example
+:code:`[node type="Camera" name="PlayerCamera" parent="Player/Head"]`
+
+Other valid keywords include:
+
+ - instance
+ - instance_placeholder
+ - owner
+ - index (if two nodes have the same name)
+ - groups
+
+The first node in the file should not have the :code:`parent=Path/To/Node` 
+entry in it's heading, and it is the scene root. All scene files should have 
+exactly one scene root. It it does not, Godot will fail to import the file. 
+The parent path of other nodes should be absolute, but without the scene 
+root's name. If it is a direct child of the scene root, it should be 
+:code:`"."`. Here is an example scene tree (but without any node content).
+::
+
+    [node name="Player" type="Spatial"]             ; The scene root
+    [node name="Arm" parent="." type="Spatial"]     ; Parented to the scene root
+    [node name="Hand" parent="Arm" type="Spatial"]
+    [node name="Finger" parent="Arm/Hand" type="Spatial"]
+    
+Similar to the internal resource, the content for each node is currently 
+undocumented. Fortunately it is easy to find out because you can simply 
+save a file with that node in it. Some example nodes are:
```

Comment:
Actually it is document -- those are just the "member" property values, saved if not default (not zero/one). Some nodes do have additional undocumented properties though.

Reply:
actually this part is now written by me, I changed the structure a little bit, so it is a copy paste from the previous document.  But I do think I should remove this part of example, since there are already or will be more detailed node document.

Reply:
>  Some nodes do have additional undocumented properties though.

yes, I write the document along with developing of TSCN exporter, so I figured out the necessary properties of each nodes and may missing some properties that are provided with default value(actually there are some essential properties provided with default value)

Reply:
When I look at it again, I decided to keep the short examples until we have a full document of these nodes